MTES-MCT/ecobalyse
# Order json

## :wrench: Problem

Json are not ordered. This create diff when the data actually doesnt change

## :cake: Solution

Order all json alphabetically and recursively. This is done 2 times for robustness :
- When generating json in python
- A check is added to block commit that don't respect this


## :desert_island: How to test

- Run `make export`
- try to commit the changes
- check that 

- edit manually one of the `activities.json` changing the order of the keys
- try to commit
- check that the json is reordered and the diff disappear
